### PR TITLE
Version Bump and Bugfixes

### PR DESCRIPTION
**Pull request by DennisKubitza**
- Bump to new Version
- Debug failing Tests
- Add Listener for Modifications on files

### CODE REVIEW

File: src/main/java/de/fraunhofer/iais/eis/ids/connector/artifact/DirectoryWatcher.java [hunk 1]
```diff
@@ -39,32 +40,46 @@ private Path createDirIfNotExists(String watchDirectory) {
...
-                                    logger.error("Could not update new artifact information at broker.", e);
+                                    logger.error("Could not delete artifact information at broker.", e);
                                     //e.printStackTrace();
                                 }
                             });
                         }
-                    }
+
+                }
+                boolean valid = key.reset();
```
Comment (maboeckmann) [inline src/main/java/de/fraunhofer/iais/eis/ids/connector/artifact/DirectoryWatcher.java]: Could you add some in-line comments here, please? What does key.reset() do?

File: src/main/java/de/fraunhofer/iais/eis/ids/connector/artifact/DirectoryWatcher.java [hunk 2]
```diff
@@ -39,32 +40,46 @@ private Path createDirIfNotExists(String watchDirectory) {
...
+                if (!valid){
```
Comment (maboeckmann) [inline src/main/java/de/fraunhofer/iais/eis/ids/connector/artifact/DirectoryWatcher.java:81]: Will this run until stopped, or is this condition met at some time?

File: src/main/java/de/fraunhofer/iais/eis/ids/connector/main/Main.java [hunk 3]
```diff
@@ -97,6 +97,9 @@
     @Value("${logging.outgoing}")
     private String loggingOut;
 
+    @Value("${jwks.trustedHosts}")
```
Comment (maboeckmann) [inline src/main/java/de/fraunhofer/iais/eis/ids/connector/main/Main.java:100]: Does this work if multiple values are provided? How are they separated - by comma?

File: src/test/resources/application-test.properties [hunk 4]
```diff
@@ -18,7 +20,7 @@ daps.keystore=resources/iais3-keystore.jks
 #daps.keystore=resources/isstbroker-keystore.jks
 daps.keystorePwd=password
 daps.keystoreAlias=1
-daps.verify=true
+daps.verify=false
```
Comment (maboeckmann) [inline src/test/resources/application-test.properties:23]: Did you intentionally turn off the DAPS validation? I guess this should be re-enabled

**Review by maboeckmann — APPROVED** [review]
Seems fine, though I would suggest turning on DAPS validation before merging. I don't know if current use cases force to turn it off.

File: src/test/resources/application-test.properties (hunk 4 above)
Comment (DennisKubitza) [reply src/test/resources/application-test.properties:23]: This wont work for the automated online tests right now. Therefore I would propose to leave this currently as false.

File: src/main/java/de/fraunhofer/iais/eis/ids/connector/main/Main.java (hunk 3 above)
Comment (DennisKubitza) [reply src/main/java/de/fraunhofer/iais/eis/ids/connector/main/Main.java:100]: As a comma seperated list, this works just like in the broker.

**Pull request merged**